### ISSUE TITLE
Create relation consistency tests for ECCVM relations

### ISSUE THREAD

**Issue by maramihali** [AztecProtocol/barretenberg#997]

**Comment by ledwards2225** [comment]
Hard to say whether this is worth it but could be a valuable tool for auditing. (The idea is to do differential testing between the real relations and versions that have been simplified as much as possible. This can add significant clarity depending on how optimized-beyond-recognition things are).

**Comment by maramihali** [comment]
Yup, currently we have something _extremely_ [basic](https://github.com/AztecProtocol/aztec-packages/blob/mm/translator-poly-fix/barretenberg/cpp/src/barretenberg/stdlib/eccvm_verifier/ecc_relation_consistency.test.cpp#L10) which I used to work through all the build issues caused by attempting to instantiate ECCVM relations on bigfield.

**Comment by notnotraju** [comment]
In principle this could be useful, but I think it's not worth the time right now.
